r/R/dplyr-summarize.R

```diff
@@ -161,6 +161,21 @@ register_bindings_aggregate <- function() {
   })
 }
 
+# we register 2 version of the "::" binding - one for use with nse_funcs
+# and another one for use with agg_funcs (below)
```

Comment:
I think what I had in mind was the location of the registration and not that of the use. I was trying to be helpful and answer the question: "Why are there 2 different bindings for `"::"`? I expanded a bit on those 2 comments, let me know if you think it could be improve.